jupiter-tests/src/test/java/org/junit/jupiter/engine/TestClassInheritanceTests.java

```diff
@@ -116,21 +116,21 @@ void beforeAndAfterMethodsInTestClassHierarchy() {
 		// @formatter:on
 
 		// @formatter:off
-		assertEquals(asList(
-			"beforeAll1",
+		assertEquals(callSequence, asList(
+				"beforeAll1",
```

Comment:
should/could undo. imho both is kind of readable as both are syncrono, thats the point actually.